channels/rdpei/client/rdpei_main.c

```diff
@@ -84,9 +86,13 @@ typedef struct
 
 	CRITICAL_SECTION lock;
 	rdpContext* rdpcontext;
+
+	BOOL async;
 	HANDLE thread;
+
 	HANDLE event;
 	BOOL running;
+	UINT64 lastPollEventTime;
```

Comment:
Just a minor comment - wrt struct packing the current layout might not be optimal - this would be better:

```
...
HANDLE event;
UINT64 lastPollEventTime;
BOOL running;
BOOL async;
```

Reply:
done